src/power_grid_model/validation/rules.py

```diff
@@ -697,6 +698,33 @@ def none_missing(data: SingleDataset, component: str, fields: Union[str, List[st
     return errors
 
 
+def valid_p_q_sigma(data: SingleDataset, component: str) -> List[MultiFieldValidationError]:
+    """
+    Check if for 'sym_power_sensor' and 'asym_power_sensor' the p_sigma and q_sigma are valid.
```

Comment:
```suggestion
    Check whether for 'sym_power_sensor' and 'asym_power_sensor' the pairs (p_sigma, q_sigma) are valid.
```

`if` is for a conditional statement (i.e.: only run the `check` in case that the pairs are valid), while `whether` implies that the statement is always run but has different result/behaviour depending on the validity of the condition.

compare e.g.:

```py
# if
if bool(condition):
   return statement()

# whether
return statement(bool(condition))
```

or in this particular case

```py
# if
if valid(p_sigma, q_sigma):
    check()

# whether
check(valid(p_sigma, q_sigma))
```

I know this is incorrect in many locations in the code and we're slowly improving, but it's better to do such a change when touching the file rather than going over all docstrings in one go, because we're bound to miss some

Reply:
Here I agree that we don't agree on if-or-not `if` is a good choice; but for the sake of eliminating ambiguity, a different way of putting it is needed. However, I don't quite agree on the second part where you distinguish `if` and `whether`. After all, English can be rather vague, and ambiguity is largely subjective...

Reply:
https://paperpal.com/blog/academic-writing-guides/language-grammar/if-vs-whether